src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -135,16 +135,16 @@ function bindCopyButton() {
 
         var isExistingSession = false;
 
-        var sessionsList = $('tr[id^="session"]');
-        if (!sessionsList.length) {
+        var $sessionsList = $('tr[id^="session"]');
+        if (!$sessionsList.length) {
             setStatusMessage(FEEDBACK_SESSION_COPY_INVALID, true);
             return false;
         }
 
-        $(sessionsList).each(function() {
-            var cells = $(this).find('td');
-            var courseId = $(cells[0]).text();
-            var feedbackSessionName = $(cells[1]).text();
+        $sessionsList.each(function() {
+            var $cells = $(this).find('td');
+            var courseId = $($cells[0]).text();
```

Comment:
Negative, jQuery wrapped objects masquerades as an array but the objects retrieved from it are not jQuery wrapped objects - .text() is a jQuery function.